prime-angular-frontend/src/app/shared/components/dialogs/content/error-logger/error-logger.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { Router } from '@angular/router';
+
+import { environment } from '@env/environment';
+import { APP_CONFIG, AppConfig } from 'app/app-config.module';
+import { DialogOptions } from '../../dialog-options.model';
+
+@Component({
+  selector: 'app-error-logger',
+  templateUrl: './error-logger.component.html',
+  styleUrls: ['./error-logger.component.scss']
+})
+export class ErrorLoggerComponent implements OnInit {
+  public error: any;
+  public enviornment = environment;
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public data: DialogOptions,
+    @Inject(APP_CONFIG) public config: AppConfig,
+    public router: Router
+  ) {
+    this.error = data.data;
```

Comment:
Although I see the same naming in `ClaimNoteComponent` e.g., can the variable `data` be called something else, possibly `options` so there isn't code like `data.data`?